test/cql-pytest/cassandra_tests/validation/operations/select_test.py

```diff
@@ -1860,7 +1860,7 @@ def dotestContainsOnPartitionKey(cql, test_keyspace, schema):
         execute(cql, table, "INSERT INTO %s (pk, ck, v) VALUES (?, ?, ?)", {5: 6}, 5, 5)
         execute(cql, table, "INSERT INTO %s (pk, ck, v) VALUES (?, ?, ?)", {7: 8}, 6, 6)
 
-        assert_invalid_message(cql, table, REQUIRES_ALLOW_FILTERING_MESSAGE,
```

Comment:
Nit: this looks like a constant that is now possibly unused.

Reply:
It is not unused. We changed (I don't know why) only the message about "allow filtering" when the filter is on the partition key - in that case we now say that you can add ALLOW FILTERING *or* change the expression on the partition key. When the filter is on a non-partition-key column, you still get the old message. This test file still uses REQUIRES_ALLOW_FILTERING_MESSAGE in many other places.

Reply:
Ok.